examples/ApiManagement/ApiManagement_Example.md

```diff
@@ -1,7 +1,6 @@
 # How To Run ApiManagement.Tests.ps1
 
-`ApiManagement.Tests.ps1` contains examples of using the `Confirm-AzBPApiManagement`, `Confirm-AzBPApiManagementApi`,
-`Confirm-AzBPApiManagementDiagnostic`, `Confirm-AzBPApiManagementLogger`, and `Confirm-AzBPApiManagementPolicy`
+`ApiManagement.Tests.ps1` contains examples of using the `Confirm-AzBPApiManagement*`
```

Comment:
Why are you removing all of the references to the cmdlets that are used in the tests?

Reply:
We do this for the cosmos cmdlets where we shorten the list of the cmldets with the `*` notation